echo-webhooks/src/main/groovy/com/netflix/spinnaker/echo/controllers/WebhooksController.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.echo.controllers
+
+import com.netflix.spinnaker.echo.events.EventPropagator
+import com.netflix.spinnaker.echo.model.Event
+import com.netflix.spinnaker.echo.model.Metadata
+import groovy.util.logging.Slf4j
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@Slf4j
+class WebhooksController {
+
+  @Autowired
+  EventPropagator propagator
+
+  @RequestMapping(value = '/webhooks/{type}/{source}', method = RequestMethod.POST)
+  void forwardEvent(@RequestBody Map postedEvent, @PathVariable String type, @PathVariable String source) {
```

Comment:
The only thing that stood out (and it's super minor) ... but for consistency across service controllers I'd suggest putting the `@RequestBody` as the last parameter rather than the first.